src/main/charts/bitbucket/values.yaml

```diff
@@ -52,6 +52,12 @@ bitbucket:
     port: 80
     # -- The type of Kubernetes service to use for Bitbucket
     type: ClusterIP
+    annotations: {}
```

Comment:
This breaks the expected output test - https://github.com/atlassian-labs/data-center-helm-charts/runs/2682877276?check_suite_focus=true

Reply:
Fixed